### ISSUE TITLE
GRM ParticleSolution errors for npar = 1

### ISSUE THREAD

**Issue by hannahlanzrath** [fau-advanced-separations/CADET-Process#160]
When setting `npar = 1`, the ParticleSolution raises `ValueError(f"Expected size:)` as the expected size of the solution does not match the solution output from CADET-Core. This is because CADET-Core provides the solution with _dimensions = (time, ncol, npar, components)_, while CADET-Process collapses the singleton dimension `npar`, expecting _dimensions = (time, ncol, components)_ when `npar = 1.`

This collapsing of the singleton dimension is still necessary in other cases. Given that choosing `npar = 1` in the GRM is not common and the LRMP would be the more rationalchoice, we could either:

TODO:
**Fix the Dimension Error:**
Adjust the handling of dimensions in ParticleSolution to properly manage the singleton dimension when npar = 1.

**Comment by schmoelder** [comment]
Could you please provide a MRE?

Also, please check whether this is solved by #185

**Comment by hannahlanzrath** [comment]
### #185 fixes the issue! 

**MRE if we ever need it** - run with branch  #188 (or #185, if it's merged) and **execute from a file in the `tests` directory.**

If the file is not part of the project, it might be necessary to comment out `from tests.create_LWE import create_lwe` in `test_cadet_adapter.py` and then additionally  `from create_LWE import *` separately to import the module. 

```
from test_cadet_adapter import *

process = create_lwe('GeneralRateModel', n_par = 1)
simulation_results = run_simulation(process)
return simulation_results
```

It will return the mentioned error.
